data/blog/introduction-to-glassmorphism-on-android.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Introduction to Glassmorphism on Android
+date: '2023-05-16'
+tags: ['ANDROID', 'KOTLIN', 'DESIGN', 'BEGINNER']
+draft: false
+summary: An introduction to glassmorphism using BlurView library
+images: []
+layout: PostLayout
+canonicalUrl:
+authors: ['othimar']
+---
+
+# Overview
+
+<TOCInline toc={props.toc} exclude="Overview" toHeading={3} />
+
+# What is glassmorphism?
+
+Glassmorphism is a design style that uses transparency and blur to create glass-like effect . For transparency, we can use a transparent color. For the blur effect, we will use **[BlurView](https://github.com/Dimezis/BlurView)** library.
```

Comment:
> Glassmorphism is a design style that uses transparency and blur to create glass-like effect .

- There is a whitespace before the full stop that should be deleted.
- `to create a glass-like effect` or  `to create glass-like effects` choose what sounds good for you.